docs/css/index.css

```diff
@@ -0,0 +1,323 @@
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
+
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: flex;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+	height: 100%;
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Helper Tags */
+
+h1 {
+	font-size: 70px;
+	font-family: 'Abril Fatface',
+		cursive;
+	text-transform: uppercase
+}
+
+h3 {
+	font-size: 24px;
+	font-family: 'Knewave',
+		cursive;
+	color: lightcoral;
+}
+
+a {
+	text-decoration: none;
+	color: white;
+}
+
+ul {
+	list-style-type: square;
+	list-style-position: inside;
+}
+
+/* End Helper Tags */
+/* Header Area */
+
+header {
+	display: flex;
+	justify-content: space-between;
+	flex-wrap: wrap;
+	top: 0;
+	width: 100%;
+	padding-top: 15px;
+	background-color: lightcoral;
+	position: fixed;
+	z-index: 5;
+}
+
+header nav {
+	height: 60px;
+	padding-top: 12px;
+	padding-left: 5%;
+}
+
+header nav a {
+	height: 30px;
+	padding: 0px 20px;
+}
+
+header nav a:hover {
+	padding: 5px 20px;
+	background-color: darkslategray;
+	color: white;
+	transition: 1s;
+}
+
```

Comment:
I'd recommend putting the `transition` effects on your selectors without the `:hover` pseudo-class. That way the transitions will apply both on and off hover!

Reply:
I'll give that a shot and see what it does, that sounds like it might have a cool effect.